cpp/daal/src/data_management/finiteness_checker.cpp

```diff
@@ -449,6 +453,7 @@ bool checkFinitenessSOAAVX512Impl(NumericTable & table, bool allowNaN)
 
     daal::threader_for_break(nCols, nCols, [&](size_t i, bool & needBreak) {
         bool * localNotFinite = tlsNotFinite.local();
+        DAAL_CHECK_MALLOC_THR(localNotFinite);
```

Comment:
After check in threading for, need make detach of `safeStat` status:
See for example [here](https://github.com/oneapi-src/oneDAL/blob/master/cpp/daal/src/algorithms/kernel_function/kernel_function_linear_dense_default_impl.i#L154).

Reply:
Fixed. Test fall are fixed too but waiting for CI to approve it